src/layouts/tables/data/projectsTableData.js

```
@@ -2,131 +2,157 @@
-            "Apca-Api-Key-Id": "PK6NYSR7W1ZRXWLX2NC9",
-            "Apca-Api-Secret-Key": "UTTc5XPc8J8qIpRVHTLmzi5ZIg30xsgy0yipL9b3",
-          },
-        }
+        }&timeframe=1M&start=${startISO}&end=${endDate}&limit=1&sort=desc`,
+        { headers }
       );
       const data = await response.json();
-      console.log(data.bars[symbol][0].c);
-      return data.bars[symbol][0].c;
+      if (data.bars && data.bars[symbol] && data.bars[symbol][0]) {
+        return data.bars[symbol][0].c;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)